.github/workflows/e2e-testing.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: npm install -g pnpm
 
       - name: Install dependencies
-        run: make install
+        run: pnpm install --frozen-lockfile
```

Comment:
Why do we have pnpm install for e2e testing, but doesn’t have for unit tests CI check ?
The source code of the CI checks should be consistent